stable/kibana/templates/deployment.yaml

```diff
@@ -40,6 +40,9 @@ spec:
         - name: "{{ $key }}"
           value: "{{ $value }}"
         {{- end }}
+        {{- range .Values.extraEnv }}
```

Comment:
best to outdent range and if calls (along with ends) as it is much more readable.